.github/workflows/python_lint.yml

```diff
@@ -1,19 +1,48 @@
-name: Python Linting
+name: Python Checks
 
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
 
+defaults:
+  run:
+    working-directory: ./python
+
 jobs:
-  lint:
+  build:
```

Comment:
Why do we need a build step?

Reply:
Just to explicitly test build and have a clear indication of a change that causes the build to break. Right now if build breaks, you'd have to dig through a test failure and see what's up, and you wouldn't necessarily know that it's a build failure